nsv13/code/game/gamemodes/overmap/objectives/rubicon.dm

```diff
@@ -8,6 +8,22 @@
 	var/system_name
 	var/datum/star_system/target_system
 
+/datum/overmap_objective/clear_system/New(datum/star_system/passed_input)
+	.=..()
+	if(passed_input)
+		system_name = passed_input.name
+	if(!system_name)
+		message_admins("No name given!")
+		for(var/datum/star_system/S in SSstar_system.neutral_zone_systems)
+			if(S.hidden)
+				continue
+			message_admins("length check")
```

Comment:
remove the debugging and it should be fine

Reply:
classic